scripts/us_bea/states_gdp/states_gdp_industry_statvars.mcf

```diff
@@ -0,0 +1,168 @@
+Node: dcid:USStateQuarterlyIndustryGDP_NAICS_11
+typeOf: dcs:StatisticalVariable
+populationType: dcs:EconomicActivity
+activitySource: dcs:GrossDomesticProduction
+measuredProperty: dcs:amount
+measurementQualifier: dcs:Nominal
+naics: dcid:NAICS/11
+
+Node: dcid:USStateQuarterlyIndustryGDP_NAICS_21
+typeOf: dcs:StatisticalVariable
+populationType: dcs:EconomicActivity
+activitySource: dcs:GrossDomesticProduction
+measuredProperty: dcs:amount
+measurementQualifier: dcs:Nominal
+naics: dcid:NAICS/21
+
+Node: dcid:USStateQuarterlyIndustryGDP_NAICS_22
+typeOf: dcs:StatisticalVariable
+populationType: dcs:EconomicActivity
+activitySource: dcs:GrossDomesticProduction
+measuredProperty: dcs:amount
+measurementQualifier: dcs:Nominal
+naics: dcid:NAICS/22
+
+Node: dcid:USStateQuarterlyIndustryGDP_NAICS_23
+typeOf: dcs:StatisticalVariable
+populationType: dcs:EconomicActivity
+activitySource: dcs:GrossDomesticProduction
+measuredProperty: dcs:amount
+measurementQualifier: dcs:Nominal
+naics: dcid:NAICS/23
+
+Node: dcid:USStateQuarterlyIndustryGDP_NAICS_31_33
+typeOf: dcs:StatisticalVariable
+populationType: dcs:EconomicActivity
+activitySource: dcs:GrossDomesticProduction
+measuredProperty: dcs:amount
+measurementQualifier: dcs:Nominal
+naics: dcid:NAICS/31_33
```

Comment:
If these are new nodes, they should include name and possibly description.

Did you do a query to see which would be new nodes? Here's an example of doing this with sparql

import datacommons as dc
s = '''
 SELECT ?dcid
 WHERE {
   ?a typeOf NAICSEnum .
   ?a dcid ?dcid
 }
'''
dc.query(s)

Reply:
To my knowledge, there has never been US state industry data in data commons, so these should all be new.

Reply:
We already have quite a few, e.g.
http://browser.datacommons.org/kg?dcid=NAICS/23
http://browser.datacommons.org/kg?dcid=NAICSEnum

That query above should return all the codes that have been previously added.

Reply:
Or did you mean that all the aggregated codes will be new?

Reply:
I thought you meant the overall node, e.g. "dcid:USStateQuarterlyIndustryGDP_NAICS_31_33." I talked to Prashanth and Tiffany yesterday and they said the convention they have followed is the same as above, except with a dash instead of an underscore, i.e. "dcid:NAICS/31-33" instead of "dcid:NAICS/31_33." Will update this and push changes.